src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,76 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size = 0;
+    private int threshold;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
Diamond operator should be used properly. It should be 'new Node<>[DEFAULT_INITIAL_CAPACITY]' instead of 'new Node[DEFAULT_INITIAL_CAPACITY]' to avoid raw type usage.